05_syntax_lab/02.py

```diff
@@ -1,6 +1,8 @@
-"""
-Write a Python program that randomizes 7 numbers
-and prints their sum total.
-If the sum is divisable by 7, also print the word "Boom"
-"""
-
+from random import randint
+
+sum = 0
+for i in range(0,7):
+    sum += randint(1,100)
+print 'Sum of 7 random numbers is:', sum
+if (sum % 7 == 0):
+    print 'Boom'
```

Comment:
No need for parenthesis in python if + one line if are usually better written as:
`if sum % 7 == 0: print 'Boom!'`